torch/_decomp/decompositions.py

```diff
@@ -3709,6 +3709,19 @@ def multilabel_margin_loss_forward(
     return z, is_target
 
 
+@register_decomposition(aten._scaled_mm)
+def _scaled_mm(mat1: Tensor, mat2: Tensor, *,
```

Comment:
just checking, should we have a decomposition for this?  What I see in this decomposition code would be an emulation of what this op is doing, but it's not actually the same thing.  I would have expected "no decomposition exists" unless we actually have primitive ops which can implement this operation exactly.